tests/test_world.py

```diff
@@ -363,6 +360,45 @@ def test_save(self):
 
             clean_temp_world(world_name_temp)
 
+        def test_get_players(self):
+            if self.world.level_wrapper.platform not in (
+                "java",
+                "bedrock",
+            ):  # Only java/bedrock platforms currently support players
+                with self.assertRaises(NotImplementedError):
+                    self.world.get_players()
+                return
+            players = [p for p in self.world.get_players()]
+            self.assertEquals(1, len(players))
+
+        def test_get_player(self):
+            if self.world.level_wrapper.platform not in (
+                "java",
+                "bedrock",
+            ):  # Only java/bedrock platforms currently support players
+                with self.assertRaises(NotImplementedError):
+                    self.world.get_players()
+                return
```

Comment:
You probably want to check if level_wrapper it is an instance of the supported classes because the platform is used to determine the format of the data. Structures such as schematic will be either Java or Bedrock based on how they were encoded.